search/cypress/support/commands.js

```diff
@@ -41,12 +41,16 @@ Cypress.Commands.add('register', (username, email, password) => {
 })
 
 Cypress.Commands.add('login', (username, password) => {
-    cy.findAllByText(/^Sign In$/).should('have.length', 1).click()
+    cy.click_text(/^Sign In$/)
     cy.get('#username')
     .type(username)
     .should('have.value', username)
     cy.get('#password')
     .type(password)
     .should('have.value', password)
     cy.findAllByRole('button', /^Sign In$/).should('have.length', 1).click()
+})
+
+Cypress.Commands.add('click_text', text => {
```

Comment:
can we use camelCase to be consistent with the rest of the library/javascript convention